packages/core-entities/src/client/yield-registry-client.ts

```diff
@@ -384,17 +384,17 @@ export class YieldRegistryClient extends ClientRegistry<YieldData> {
         const underlying = tokens.getTokenByID(network, v.underlying);
         const { defaultLeverageRatio, maxLeverageRatio } =
           config.getVaultLeverageFactors(network, v.vaultAddress);
-        const vaultAPYs = (analytics
+        /*   const vaultAPYs = (analytics
           .getVault(network, v.vaultAddress)
           ?.filter(
             ({ timestamp }) => timestamp > getNowSeconds() - 7 * SECONDS_IN_DAY
           )
           .map(({ totalAPY }) => totalAPY)
-          .filter((apy) => apy !== null) || []) as number[];
-        const totalAPY =
-          vaultAPYs.length > 0
-            ? vaultAPYs.reduce((t, a) => t + a, 0) / vaultAPYs.length
-            : 0;
+          .filter((apy) => apy !== null) || []) as number[]; */
+        const totalAPY = 0;
```

Comment:
Similarly here, maybe we can add some sort of environment flag that gets set in the backfill job and then we can turn off this code if it is set?